pypeit/specobjs.py

```diff
@@ -506,23 +517,35 @@ def __repr__(self):
     def __len__(self):
         return len(self.specobjs)
 
-    def build_header(self, head_fitstbl, spectrograph):
+    def build_header(self, head_fitstbl, head2d, spectrograph):
         """
         Builds the spec1d file header from the fitstbl meta data and meta data from spectrograph
+        and meta data from original header
+
         Args:
-            head_fitstbl (header):
+            head_fitstbl (:class:`astropy.io.fits.Header`):
```

Comment:
Just a heads-up: For stuff that's outside pypeit, all the `:class:` prefix does is set the font for the class path.  For this kind of thing, I tend to do:
```
head_fitstbl (`astropy.io.fits.Header`_)
```
where the type is now a link defined in `pypeit/doc/links.rst`.  At the top of the file you'll see the lines:
```
.. include:: ../links.rst
```
which includes the link definitions for all docstrings in the module.